src/components/form/form.jsx

```diff
@@ -128,3 +132,74 @@ Form.Feedback = ({ className, message }) => {
     </>
   );
 };
+
+Form.Combobox = ({ id, labelText, className, list, name, nameKey, idKey }) => {
+  const [selectedItems, setSelectedItems] = useState([]);
+  const [query, setQuery] = useState("");
+  const filteredList =
+    query === ""
+      ? list
+      : list.filter((item) => {
+          return item[nameKey].toLowerCase().includes(query.toLowerCase());
+        });
+  return (
+    <div className={`flex font-bold font-arial flex-col py-3 ${className}`}>
+      <label htmlFor={id} className={`mb-1.5 text-lg text-start `}>
+        {labelText}
+      </label>
+      <Combobox
+        value={selectedItems}
+        onChange={setSelectedItems}
+        multiple
+        nullable
+        name={name}
+      >
+        <div className=" relative font-normal py-3 px-4 border-solid rounded-none border border-black focus-within:ring-black focus-within:ring-1 flex gap-2.5 flex-wrap">
+          {selectedItems.length > 0 &&
+            selectedItems.map((item) => (
+              <div
+                key={item[idKey]}
+                className="flex gap-1.5 bg-neutral-300 px-1.5 py-1"
+              >
+                <span>{item[nameKey]}</span>
+                <button
+                  type="click"
+                  onClick={() => {
```

Comment:
I think this onClick handler would be better to add to the entire input, so that the user can see the items when they interact with the input.

Reply:
I'm gonna need a little clarification here. If I'm not mistaken, this is the button that deselects the item from the selected list.
So you want the user to be able to deselect by clicking on the entire thing and not just the button?

Reply:
I'm responding on mobile and can't see all of the code as this comment box is covering it. If this is the button to deselect an item then I was mistaken.

What I meant was that the options should display when the user clicks on the input of the combo box and not just when they click on the drop down button on the right. I believe this is how most combo boxes operate across the web and would be more intuitive.